app/src/test/java/com/ichi2/apisample/MusIntervalTest.java

```diff
@@ -56,7 +64,7 @@ public void checkExistence_NoSuchStartingNote() throws AnkiDroidHelper.InvalidAn
         doReturn(modelId).when(helper).findModelIdByName(defaultModelName);
         doReturn(SIGNATURE).when(helper).getFieldList(eq(modelId));
         doReturn(deckId).when(helper).findDeckIdByName(defaultDeckName);
-        doReturn(existingNotesData).when(helper).findNotes(eq(modelId), any(Map[].class));
+        doReturn(existingNotesData).when(helper).findNotes(eq(modelId), any(ArrayList.class));
```

Comment:
Can we remove "@SuppressWarnings("unchecked")" from test definition after this change?
If yes, please do it too (and similar changes below).

Reply:
In this case, there still is an unchecked conversion from ArrayList to ArrayList<Map<String, String>>